cfy_manager/utils/certificates.py

```diff
@@ -18,14 +18,22 @@
 )
 from ..exceptions import ProcessExecutionError
 from .files import write, write_to_tempfile, remove
-from ..components.validations import check_certificates, validate_certificates
 
 from ..logger import get_logger, setup_console_logger
 from .. import constants as const
+from cfy_manager.exceptions import ValidationError
 
 logger = get_logger('Certificates')
 
 
+def get_cert_cn(cert_path):
+    raw = run(
+        ['openssl', 'x509', '-noout', '-subject', '-in', cert_path]
+    ).aggr_stdout
+    # The raw value will be something like "subject=CN = *.cloudify.co"
+    return raw.split('=')[-1].strip()
```

Comment:
`CN` isn't always going to be the only value there, eg.
```
➜  ~ openssl x509 -noout -subject -in /etc/ssl/certs/DigiCert_Global_Root_CA.pem 
subject=C = US, O = DigiCert Inc, OU = www.digicert.com, CN = DigiCert Global Root CA
```

Reply:
Good point. I'm uncertain if it'll always /end/ with CN, but just in case it doesn't I'll tweak this.